app/src/main/kotlin/com/hippo/ehviewer/ui/reader/PagerViewer.kt

```diff
@@ -259,4 +261,7 @@ private inline fun ZoomableState?.canPan(getRemaining: (Rect) -> Float): Boolean
     return this != null && Settings.navigateToPan.value && getRemaining(transformedContentBounds) > 1f
 }
 
-private val PagerZoomSpec = ZoomSpec(maxZoomFactor = 5f)
+private val PagerZoomSpec = ZoomSpec(
+    maximum = ZoomLimit(factor = 5f),
+    minimum = ZoomLimit(factor = 1f, overzoomEffect = OverzoomEffect.Disabled),
```

Comment:
It doesn't work actually.
https://github.com/saket/telephoto/blob/62bd3977351eb6f6d8742a1940d64127da0f3f5c/zoomable/src/commonMain/kotlin/me/saket/telephoto/zoomable/RealZoomableState.kt#L244
Only the `maximum.overzoomEffect` is used.